privacyidea/lib/machineresolver.py

```diff
@@ -111,7 +112,7 @@ def save_resolver(params):
     # create the config
     for key, value in data.items():
         if types.get(key) == "password":
-            value = encryptPassword(value)
+            value = binascii.hexlify(encryptPassword(value))
```

Comment:
Hmm, since we're going to store the password in a Unicode column, should we maybe use ``hexlify_and_unicode``? @plettich what do you think?

Reply:
makes sense if it is stored in the db. Unfortunately (un-)hexlify, base64(de/en)code, etc all return bytes instead of strings so we have to decode them.

OT: how can i tell github to notify me if someone mentions me?

Reply:
I am currently reworking the ``encryptPasswort`` and ``decryptPasswort`` so that they will always take and return unicode. So this will change... I will adapt this after my other PR.

Reply:
fixed in 40beef11